MovieApp/MovieApp/App/Features/Movies/Components/MainInfoView/MainInfoView+Design.swift

```diff
@@ -0,0 +1,119 @@
+import UIKit
+import SnapKit
+
+extension MainInfoView: DesignProtocol {
+    
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    func createViews() {
+        moviePoster = UIImageView()
+        addSubview(moviePoster)
+        
+        backgroundProgressPath = CAShapeLayer()
+        layer.addSublayer(backgroundProgressPath)
+        
+        progressBar = CAShapeLayer()
+        layer.addSublayer(progressBar)
+        
+        progressLabel = UILabel(with: .medium(size: 15))
+        addSubview(progressLabel)
```

Comment:
Is there a reason you are not putting subviews inside moviePoster?

Reply:
code looked cleaner without putting subviews inside moviePoster, but it works either way.
I will put all subviews inside moviePoster if that's better practice.

Reply:
I really don't know. Just asking if you found out. I think I'm the only one doing that